src/commands/sdm.js

```diff
@@ -1,10 +1,10 @@
 "use strict";
 
 // Dependencies
-let moment = require("moment");
+const moment = require("moment");
```

Comment:
[`let` over `const` for `require()`](https://github.com/NullDevCo/JavaScript-Styleguide#references--let-require)